content/user/ui/report/Org-stats.md

```diff
@@ -0,0 +1,51 @@
+
+<a href="assets/images/org-stats-reports-page.png" rel="lightbox" title="Organization statistics on Reports page"><img class="thumbnail" src="assets/images/org-stats-reports-page.png"/></a>
+
+In the Licenses chart, view the number of overall licenses for Assess and Protect, as well as the the number of unlicensed applications and servers that exist in your organization. Click the application count to go to the Trial/Unlicensed quick view in the Applications page. 
+
+In the next chart, view active applications. The inner ring designates the breakdown by language; choose the breakdown of the outer ring by selecting **Technology** or **Grade** in the dropdown menu. 
+
+Last, but not least, view the Servers chart. Select **Container** or **Environment** in the dropdown menu to choose how the numbers are analyzed. 
+
+For more details on each category, select the **View** link under each heading. 
+
+## Licenses
+
+The Licenses tab features a Consumption trend chart of data for the past 12 months. Hover over a data point on the Assessment or Protection trend lines to see how may licenses were available that month. The thermometer chart below shows the total number of licenses purchased compared to the number being used. The timeline of shows how many licenses are about to expire on given dates. For a different view of the data, take a look at the circular charts with breakdowns by fraction and percentage for Assessment and Protection. 
+
+If you have no Protection licenses, Contrast alerts you to the count of unlicensed servers.
+
+<a href="assets/images/org-stats-licenses.png" rel="lightbox" title="Organization statistics for licenses"><img class="thumbnail" src="assets/images/org-stats-licenses.png"/></a>
+
+<!-- Couldn't fit Assessment AND Protection view in browser window-->
+
+## Applications
+
+In the Applications tab, the Status thermometer chart shows the total number of applications broken down by the number that are licensed, unlicensed and archived. Click on the total number of onboarded applications to go to the Applications page for more details on each one. Contrast also remdinds you of how many licenses are available in your organization. 
+
+The circular Language Breakdown chart shows the number of applications by language in the inner band and by Technology or Grade in the outer band. Click the number of active applications to go to the grid in the Applications page. High Risk, Monitored and Expirations snapshots show the number of applications with critical open vulnerabilities, no protection and expiring licenses, respectively. Applications that you added within the last week and applications in an offline server are listed separately in the sidebar. 
```

Comment:
"Click on the active applications count to go to move directly to the Applications page grid......Applications added within the last week as well as applications that reside on an offline server are..."